src/components/scenes/SendScene2.tsx

```diff
@@ -948,28 +950,41 @@ const SendComponent = (props: Props) => {
     disabledText = lstrings.spending_limits_enter_pin
   }
   return (
-    <SceneWrapper background="theme">
-      <KeyboardAwareScrollView contentContainerStyle={styles.contentContainerStyle} extraScrollHeight={theme.rem(2.75)} enableOnAndroid>
-        {renderSelectedWallet()}
-        {renderAddressAmountPairs()}
-        {renderAddAddress()}
-        {renderTimeout()}
-        {renderError()}
-        {renderFees()}
-        {renderMetadataNotes()}
-        {renderSelectFioAddress()}
-        {renderUniqueIdentifier()}
-        {renderInfoTiles()}
-        {renderAuthentication()}
-        {renderScamWarning()}
-      </KeyboardAwareScrollView>
-      <View style={styles.footer}>
-        {showSlider && <SafeSlider disabledText={disabledText} onSlidingComplete={handleSliderComplete} disabled={disableSlider} />}
-      </View>
+    <SceneWrapper background="theme" hasNotifications hasOverscroll={false}>
+      {gap => (
+        <>
+          <KeyboardAwareScrollView contentContainerStyle={{ paddingBottom: gap.bottom }} extraScrollHeight={theme.rem(2.75)} enableOnAndroid>
```

Comment:
`style={ marginBottom: -gap.bottom }`

Reply:
No this is right, it should be positive. The notificationHeight is meant as an
contentContainerStyle paddingBottom (overscroll) - using it as a style padding causes a visual cutoff